Build fail real:

```
======================================================================
FAIL: test_instantiate_all (testapps.test_pillowtop.tests.test_settings.PillowtopSettingsTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/mnt/testapps/test_pillowtop/tests/test_settings.py", line 33, in test_instantiate_all
    self.assertEqual(expected_meta[pillow_config.name], _pillow_meta_from_config(pillow_config))
```

--- a/testapps/test_pillowtop/tests/data/all-pillow-meta.json
+++ b/testapps/test_pillowtop/tests/data/all-pillow-meta.json
@@ -600,21 +600,21 @@
         "name": "UserGroupsDbKafkaPillow"
     },
     "UserPillow": {
-        "advertised_name": "corehq.pillows.user.UserPillow.test_hqusers_2016-01-20_1610.testhq",
+        "advertised_name": "corehq.pillows.user.UserPillow.test_hqusers_2016-02-16_1402.testhq",
         "change_feed_type": "CouchChangeFeed",
-        "checkpoint_id": "corehq.pillows.user.UserPillow.test_hqusers_2016-01-20_1610.testhq",
+        "checkpoint_id": "corehq.pillows.user.UserPillow.test_hqusers_2016-02-16_1402.testhq",
         "couch_filter": "users/all_users",
         "couchdb_type": "Database",
         "couchdb_uri": "http://{COUCH_SERVER_ROOT}/test_commcarehq__users",
         "document_class": "CommCareUser",
         "es_alias": "hqusers",
-        "es_index": "test_hqusers_2016-01-20_1610",
+        "es_index": "test_hqusers_2016-02-16_1402",
         "es_type": "user",
         "extra_args": {},
         "full_class_name": "corehq.pillows.user.UserPillow",
         "include_docs": true,
         "name": "UserPillow",
-        "unique_id": "test_hqusers_2016-01-20_1610"
+        "unique_id": "test_hqusers_2016-02-16_1402"
     },
     "WorldVisionChildFluffPillow": {
         "advertised_name": "fluff.WorldVisionChildFluffPillow.testhq",

